src/main/java/org/eclipse/tractusx/ssi/lib/validation/JsonLdValidatorImpl.java

```diff
@@ -31,7 +31,7 @@
 import jakarta.json.JsonObject;
 import jakarta.json.JsonValue;
 import java.util.Map;
-import org.eclipse.tractusx.ssi.lib.exception.InvalidJsonLdException;
+import org.eclipse.tractusx.ssi.lib.exception.json.InvalidJsonLdException;
```

Comment:
When the JsonLdError is caught (lin 78), it should rather generate a message for the respective JsonLdErrorCode (using e.g. JsonLdError.getCode().toMessage()) instead of just wrapping the original exception.